tests/Fixture/ExternalEmployeesFixture.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace SwaggerBake\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+/**
+ * ExternalEmployeesFixture
+ */
+class ExternalEmployeesFixture extends TestFixture
+{
+    public $connection = 'test_alt';    
```

Comment:
The whole point of this fixture is so that it can specify that it goes with an alternate connection. Let me know if there's another (existing) fixture you'd rather I move this to.